docs/getting-started/quick-start-guide.asciidoc

```diff
@@ -0,0 +1,135 @@
++
+[source,text]
+products.taxless_price >= 60 AND category : Women's Clothing
++
+The query returns the women's clothing orders for $60 and more.
++
+[role="screenshot"]
+image::images/tutorial-discover-4.png[]
+
+. Hover over the list of *Available fields*, then click *+* next to the fields you want to view in the table.
++
+For example, when you add the *category* field, the table displays the product categories for the orders.
++
+[role="screenshot"]
+image::images/tutorial-discover-3.png[]
+
+For more information, refer to <<discover, *Discover*>>.
+
+[float]
+[[view-and-analyze-the-data]]
+== View and analyze the data
+
+Gain insight into your data with *Dashboard*, which is a collection of panels that you can 
+use to view and analyze your data. Panels contain visualizations, interactive controls, Markdown, and more.
+
+. Open the menu, then click *Dashboard*. 
+
+. Click *[eCommerce] Revenue Dashboard*.
+
+[role="screenshot"]
+image::getting-started/images/tutorial-sample-dashboard.png[]
+
+[float]
+[[filter-and-query-the-data]]
+=== Filter the data
+
+To focus in on the data you want to view on the dashboard, use filters. 
+
+. From the *Controls* visualization, make a selection from the *Manufacturer* and *Category* dropdowns, then click *Apply changes*.
++
+For example, the following dashboard shows the data for women's clothing from Gnomehouse. 
++
+[role="screenshot"]
+image::getting-started/images/tutorial-sample-filter.png[]
+
+. To manually add a filter, click *Add filter*, then specify the options.
++
+For example, to view the orders for Wednesday, select *day_of_week* from the *Field* dropdown, select *is* from the *Operator* dropdown, then select *Wednesday* from the *Value* dropdown.
++
+[role="screenshot"]
+image::getting-started/images/tutorial-sample-filter2.png[]
+
+. When you are done, remove the filters.
+
+For more information, refer to <<dashboard,*Dashboard*>>.
+
+[float]
+[[create-a-visualization]]
+== Create a visualization
+
+Use *Lens* to create a treemap that shows the top regions and manufacturersthen, then add it to the dashboard.
+
+. From the {kib} toolbar, click *Edit*, then click *Create new*.
+
+. On the *New Visualization* window, click *Lens*.
+
+. From the *Available fields* list, drag and drop the following fields to the visualization builder:
+
+* *geoip.city_name*
+
+* *manufacturer.keyword*
++
+[role="screenshot"]
+image::getting-started/images/tutorial-treemap.png[Treemap in Lens that shows the top regions and manufacturers from the Ecommerce sample data]
+
+. Click *Save, enter a title, then click *Save and return*.
++
+For more information, refer to <<lens, *Lens*>>.
+
+[float]
+[[quick-start-next-steps]]
+== Next steps
+
+If you are you ready to add your own data, refer to <<connect-to-elasticsearch,Add data to {kib}>>.
+
+If you want to ingest your data, refer to <<ingest-manager, Ingest Manager>>.
```

Comment:
This might be a better link for Ingest Manager:  https://www.elastic.co/guide/en/ingest-management/current/ingest-management-getting-started.html